packages/backend/src/index.ts

```diff
@@ -28,28 +28,38 @@ import {
   notFoundHandler,
   requestLoggingHandler,
 } from '@backstage/backend-common';
-import { router as inventoryRouter } from '@backstage/plugin-inventory-backend';
+import { createRouter as inventoryRouter } from '@backstage/plugin-inventory-backend';
 import compression from 'compression';
 import cors from 'cors';
 import express from 'express';
 import helmet from 'helmet';
 import { testRouter } from './test';
 
 const DEFAULT_PORT = 7000;
-
 const PORT = parseInt(process.env.PORT ?? '', 10) || DEFAULT_PORT;
-const app = express();
+const logger = getRootLogger().child({ type: 'plugin' });
+
+async function main() {
+  const app = express();
+
+  app.use(helmet());
+  app.use(cors());
+  app.use(compression());
+  app.use(express.json());
+  app.use(requestLoggingHandler());
+  app.use('/test', testRouter);
+  app.use(
+    '/inventory',
+    await inventoryRouter({
+      logger,
+    }),
+  );
```

Comment:
I like this. do you envision having GraphQL composition happen in here too, should a user want it?

Reply:
Yeah.

As a matter of fact, as a next step, this is how it could look for Spotify:

```
const inventory = new AggregatorInventory();
inventory.enlist(new SysmodelInventory()); // until the open source version is done?
inventory.enlist(new StyxInventory()); // or whatever one wants

app.use('/inventory', await inventoryRouter({ logger, inventory }));

// now also call the inventory directly from graphql resolvers!
```

Wiring is supposed to happen in here, not inside inventor-backend which is supposed to be a fixed, reusable module across orgs :)